sentry_sdk/integrations/django/__init__.py

```diff
@@ -701,8 +701,14 @@ def _set_db_data(span, cursor_or_db):
         connection_params = cursor_or_db.connection.get_dsn_parameters()
     else:
         try:
-            # psycopg3
-            connection_params = cursor_or_db.connection.info.get_parameters()
+            # psycopg3, only extract needed params as get_parameters
+            # can be slow because of the additional logic to filter out default
+            # values
```

Comment:
```suggestion
            # psycopg3, extract current connection params as get_parameters
            # can be slow because of the additional logic to filter out default
            # values
```